### PR TITLE
Address OAB review comments.

### PR DESCRIPTION
**Pull request by pvretano**
Address review comments from the OAB before RFC release.

### CODE REVIEW

File: core/standard/clause_7_building_blocks.adoc [hunk 1]
```diff
@@ -909,6 +911,14 @@ sort order.
 
 include::requirements/sorting/REQ_defaultSortOrder-definition.adoc[]
 
+==== Sorting by relevance (informative)
+
+A common practice is to sort search results by relevance.
+
+This capability is accomodated in this Standard by using <<per_sorting_sortables-synthetic,synthetic>> sortables.  That is, sortables that are derived or synthesized by the server.
```
Comment (tomkralidis) [inline core/standard/clause_7_building_blocks.adoc:918]: -> This capability is accommodated in this Standard by using <<per_sorting_sortables-synthetic,synthetic>> sortables.  That is, sortables that are derived or synthesized at the server's discretion.

File: core/standard/clause_7_building_blocks.adoc [hunk 2]
```diff
@@ -909,6 +911,14 @@ sort order.
...
+
+A sever implementing relevance sorting might include, in the list of sortables, a sortable named `relevance` that is dynamically computed for each result in a result set.  The value of the synthetic `relevance` sortable could, for example, be between 0 and 10 with zero indicating least relevance and 10 indicating most relevance.  A `sortBy` clause specified on a request specifying a descending sort be the `relevance` property would present search results from most to least relevant.
```
Comment (tomkralidis) [inline core/standard/clause_7_building_blocks.adoc:920]: A server implementing relevance ranking might include, in the list of sortables, a sortable named `relevance` that is dynamically computed for each result in a result set.  The value of the synthetic `relevance` sortable could, for example, be between 0 and 10 with zero indicating least relevance and 10 indicating most relevance.  A `sortby` clause specified on a request specifying a descending sort be the `relevance` property would present search results from most to least relevant.

File: core/examples/json/sortables.json [hunk 3]
```diff
@@ -19,6 +19,14 @@
          "description": "This is a synthetic sortable that the server computes on the fly based on the spatial extent of the resource.  Resources without spatial extent have zero area.",
          "type": "number"
       },
+      "relevance": {
+         "title": "Relevance of the result",
+         "description": "This is a synthetic sortable that the server computes on the fly to assign a relevance to the result with respect to the query parameters specified in the original request",
```
Comment (tomkralidis) [inline core/examples/json/sortables.json:24]: -> This is a synthetic sortable at the server's discretion that the server dynamically computes to determine relevance to the result with respect to the query parameters specified in the original request

**Review by tomkralidis — CHANGES_REQUESTED** [review]

File: core/examples/json/sortables.json (hunk 3 above)
Comment (pvretano) [reply core/examples/json/sortables.json:24]: @tomkralidis are you asking to add "at the server's discretion"?  Why?  All sortables are at the server's discresion.  That is to say that the server decides what is a sortable or not and then create the respond accordingly.

Comment (tomkralidis) [reply core/examples/json/sortables.json:24]: Yes, however sorting alphabetically or numerically is more predictable than a full text search.  I guess the main point here is that for relevance ranking, the actual algorithm to rank and provide a "score" is, as mentioned today on OAB, "magic".

Comment (kalxas) [reply core/examples/json/sortables.json:24]: I do not think we need to clarify more.

Comment (pvretano) [reply core/examples/json/sortables.json:24]: The "magic" is how the value of computed and that is not relevant here just that is computed and exposed as a sortable to allow the results to be ordered accoring to the relevance ranking.  Are you trying to say that the computation is at the discration of the server?  If yes, then I can reword a bit to convey that.

**Review by kalxas — APPROVED** [review]

File: core/standard/clause_7_building_blocks.adoc (hunk 1 above)
Comment (kalxas) [reply core/standard/clause_7_building_blocks.adoc:918]: I am ok to leave as is

File: core/examples/json/sortables.json (hunk 3 above)
Comment (tomkralidis) [reply core/examples/json/sortables.json:24]: Works for me, +1

**Review by kalxas — APPROVED** [review]

**Pull request merged**